pkg/chunk/cache/fifo_cache.go

```diff
@@ -213,6 +244,9 @@ func (c *FifoCache) put(ctx context.Context, key string, value interface{}) {
 	c.entries[c.last].next = index
 	c.first = index
 	c.index[key] = index
+
+	c.memoryBytes.Add(float64(sizeOf(key) + sizeOf(value)))
+	c.entriesCurrent.Set(float64(len(c.entries)))
```

Comment:
[nit, no strong opinions] Is it equal to do `c.entriesCurrent.Inc()` or am I wrong? If I'm correct, `.Inc()` may be just clearer.